locales/en.json

```diff
@@ -237,5 +237,17 @@
     "yunohost_ca_creation_failed": "Unable to create certificate authority",
     "yunohost_configured": "YunoHost has been configured",
     "yunohost_installing": "Installing YunoHost...",
-    "yunohost_not_installed": "YunoHost is not or not correctly installed. Please execute 'yunohost tools postinstall'."
+    "yunohost_not_installed": "YunoHost is not or not correctly installed. Please execute 'yunohost tools postinstall'.",
+    "ask_alias_name" : "Alias name",
+    "ask_alias_mailforward" : "List of email addresses to forward to",
+    "pattern_listemail" : "Must be a valid email address list delimited by commas",
+    "alias_init" : "Initializing aliases in LDAP",
+    "alias_created" : "Alias successfully created",
+    "alias_creation_failed" : "Unable to create alias",
+    "alias_deletion_failed" : "Unable to delete alias",
+    "alias_deleted" : "Alias successfully deleted",
+    "alias_unknown" : "Unknown alias",
+    "alias_updated" : "Alias successfully updated",
+    "alias_update_failed" : "Unable to update alias",
+    "alias_info_failed" : "Unable to retrieve alias information"
```

Comment:
are we required to lexicographically order this file ? or is this done by weblate ?